src/index.js

```diff
@@ -1,47 +1,83 @@
 import './index.css';
 
-// Get DOM
-const listElement = document.createElement('div');
-listElement.classList = 'dynamic-container';
+let todoList = window.localStorage.getItem('todoList') ? JSON.parse(localStorage.getItem('todoList')) : [];
+const inputItem = document.querySelector('.inputs-field');
 const listContainer = document.querySelector('.list');
+const addButton = document.querySelector('.add-btn');
+const clearAll = document.querySelector('#clear-btn');
 
-const task = [
-  {
-    description: 'task1',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'task2',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'task3',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'task4',
-    completed: false,
-    index: 4,
-  },
-];
-function iterateTasks() {
-  task.forEach((item) => {
-    listElement.innerHTML
-        += `
-        <div class="list-properties">
-            <span class="left-items">
-                <input type="checkbox">
-                <p>${item.description}</p>
-            </span>
-            <i class="fa-solid fa-ellipsis-vertical"></i>
-        </div>
-        <hr>
-        `;
+const displayToDoList = () => {
+  listContainer.innerHTML = '';
+  todoList.forEach((item, index) => {
+    const listItem = document.createElement('div');
+    listItem.className = 'list-properties';
+    listItem.innerHTML = `
+    <div class="both-sides">
+      <span class="left-items">
+        <input type="checkbox" ${item.completed ? 'checked' : ''}>
+        <p>${item.description}</p>
+      </span>
+      <span class="right-items">
+        <div id="remove-single"><i class="fa-solid fa-trash"></i></div>
+        <i class="fa-solid fa-ellipsis-vertical"></i>
+      </span>
+      </div>
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if this method displayTodoList ran when the dom loads 👍 because when your app runs for the first time then we don't see any todo we need to add one todo to see all the other todos 😨 To fix this kindly add a code like below ⬇️ at the bottom of your javascript file 💯

`window.onload = displayToDoList();`